virtual-setup/zoom-procedures.md

```diff
@@ -1 +1,197 @@
+* [Windows](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/windows-instructions.md#zoom)
+
+#### Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [General tips for using Zoom during workshops](#general-tips-for-using-zoom-during-workshops)
+  - [Audio](#audio)
+  - [Video](#video)
+  - [Interacting with instructors and providing feedback](#interacting-with-instructors-and-providing-feedback)
+    - [Participant pane](#participant-pane)
+    - [Chat](#chat)
+- [Joining a Zoom call](#joining-a-zoom-call)
+  - [To join a call](#to-join-a-call)
+- [Zoom recordings](#zoom-recordings)
+- [Using Zoom breakout rooms](#using-zoom-breakout-rooms)
+  - [Asking for 1:1 help during instruction](#asking-for-11-help-during-instruction)
+  - [Joining a breakout room](#joining-a-breakout-room)
+  - [Sharing your screen in Zoom breakout rooms](#sharing-your-screen-in-zoom-breakout-rooms)
+  - [Returning to the main session](#returning-to-the-main-session)
+  - [Consultation days](#consultation-days)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## General tips for using Zoom during workshops
+
+### Audio
+
+Please make sure your microphone is muted when joining the call and when you are to minimize background noise during instruction.
+You can toggle mute on and off using the `Mute` (or `Unmute` if you are muted) icon on the left side of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-mute.png>
+
+Where possible, please utilize headphones throughout training to minimize audio feedback.
+
+### Video
+
+If you are experiencing issues with the call quality, turning off your video may help.
+You can toggle video on and off using the Stop Video icon (or Start Video if video is off) on the left side of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-video.png>
+
+### Interacting with instructors and providing feedback
+
+#### Participant pane
+
+The participant pane will both allow you to identify the host and co-hosts of the meeting (CCDL staff) and to provide feedback during instruction. 
+
+To bring up the participant pane, click the participants icon in the middle of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-participant-icon.png>
+
+The participant pane will appear on the side of the main Zoom window and look something like this:
+
+<img src = screenshots/zoom-participant-panel.png height = 500>
+
+There are feedback controls on the bottom of the participant pane that will allow you to answer yes or no questions, give feedback on the speed of instruction, and under `more` you can indicate if you need a break.
+
+<img src = screenshots/zoom-participant-feedback.png width = 400>
```

Comment:
This image (and the text above) does not include the "raise hand" option. I do think we want to mention it, as we might ask people to raise hands when they finish a step.

`zoom-chat-pane.png` has the set of icons they will see.